testpilot-metrics.js

```diff
@@ -0,0 +1,333 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Usually, what I think we should do when we take an external file, is to have a separate commit _just_ with this file. and then in separate commits we start using it.